installer/PacketFence-Forwarder.nsi

```diff
@@ -59,33 +59,26 @@ Function .onInit
   StrCpy $AppDataPath "$1"
 FunctionEnd
 
-Section "Winpcap Installer" SEC01

```

Comment:
So, we do not test anymore if npcap is installed, correct?

Reply:
No

Reply:
So, could it be a good idea to test if it is available during the install?

Reply:
I have my answer here #d15feb6f73270b7a43f38426952355e893243397